src/main/java/com/github/takayoshi24/cinema/reservation/ReservationController.java

```diff
@@ -0,0 +1,50 @@
+package com.github.takayoshi24.cinema.reservation;
+
+import com.github.takayoshi24.cinema.seans.Seans;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+public class ReservationController {
+
+    private final ReservationService reservationService;
+
+    @Autowired
+    public ReservationController(ReservationService reservationService) {
+        this.reservationService = reservationService;
+    }
+
+    @GetMapping(params = {"email"})
+    public Page<Reservation> getReservationsByEmail(@RequestParam String  email, Pageable pageable){
+        return reservationService.getReservationsByEmail(email, pageable);
+    }
+
+    @GetMapping
+    public List<Reservation> getReservations(){
+        return reservationService.getReservations();
+    }
```

Comment:
Use pagination.
Is this endpoint useful? What's the goal of listing reservations with no Seance boundary?